descriptors/ModuleDescriptor-template.json

```diff
@@ -61,6 +61,13 @@
           ],
           "pathPattern": "/search/instances/facets",
           "permissionsRequired": [ "search.instances.facets.collection.get" ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/search/instances/ids",
+          "permissionsRequired": [ "search.instances.ids.collections.get" ]
```

Comment:
Do you mean `collections` or just `collection`? Other similar perms used the latter.